### PR TITLE
ensure the AI Company ID to Entity ID data has only distinct rows

### LINKED ISSUE #101: deal with `left_join()` many-to-many relationship warnings
A few `left_join()` commands in the abcd_flags section of [run_pacta_data_preparation.R](https://github.com/RMI-PACTA/workflow.data.preparation/blob/main/run_pacta_data_preparation.R) throw many-to-many relationship warnings.

In each case, we should verify if the many-to-many relationship is truly expected (conceptually). If so, these warnings can be silenced by explicitly specifying `relationship = "many-to-many"`. If not, we need to look into why that is happening and fix it.

I have identified the following lines that exhibit this behavior:

https://github.com/RMI-PACTA/workflow.data.preparation/blob/c08b8fb5a139716997ae2c0e83a01757d51a9a2d/run_pacta_data_preparation.R#L359

https://github.com/RMI-PACTA/workflow.data.preparation/blob/c08b8fb5a139716997ae2c0e83a01757d51a9a2d/run_pacta_data_preparation.R#L360

https://github.com/RMI-PACTA/workflow.data.preparation/blob/c08b8fb5a139716997ae2c0e83a01757d51a9a2d/run_pacta_data_preparation.R#L384

https://github.com/RMI-PACTA/workflow.data.preparation/blob/c08b8fb5a139716997ae2c0e83a01757d51a9a2d/run_pacta_data_preparation.R#L385

https://github.com/RMI-PACTA/workflow.data.preparation/blob/c08b8fb5a139716997ae2c0e83a01757d51a9a2d/run_pacta_data_preparation.R#L389-L392

### PR DESCRIPTION
**Pull request by cjyetman**
closes #101

### CODE REVIEW

File: run_pacta_data_preparation.R [hunk 1]
```diff
@@ -243,7 +243,8 @@ factset_entity_id__ar_company_id <-
   select(
     factset_entity_id = "factset_id",
     ar_company_id = "company_id"
-  )
+  ) %>%
+  distinct()
```
Comment (jdhoffa) [inline run_pacta_data_preparation.R:247]: NIT: Should this be something we warn about? like "X input contains duplicate rows, removing"
Maybe not important.

**Review by jdhoffa — APPROVED** [review]
approved with nit comment

File: run_pacta_data_preparation.R (hunk 1 above)
Comment (cjyetman) [reply run_pacta_data_preparation.R:247]: 🤷 maybe? I think that it should be expected that all of these inputs do not have random duplicate lines. 

I've already gone down the rabbit hole of imaging adding validation functions to pacta.data.preparation for all of these inputs and all of the configuration options 😈

Comment (jdhoffa) [reply run_pacta_data_preparation.R:247]: yeah i mean, in my perfect world, once https://github.com/RMI-PACTA/workflow.data.preparation/issues/94 is closed, I would imagine just adding a simple "warning" message in the wrapper function whenever this kind of wonkiness happens would be sufficient

**Pull request merged**